### PR TITLE
Desigate 1

### PR DESCRIPTION
**Pull request by dkehn**

### CODE REVIEW

**Review by dprince — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1150,6 +1206,24 @@ octavia_kuttl: input openstack_crds deploy_cleanup mariadb mariadb_deploy keysto
 	make keystone_cleanup
 	make mariadb_cleanup
 
+.PHONY: octavia_kuttl_run
```
Comment (rabi) [inline Makefile:1207]: Is this relevant to this PR? Seems like duplicated target.

Comment (Sandeepyadav93) [reply Makefile:1207]: @dkehn I have proposed a PR: https://github.com/openstack-k8s-operators/install_yamls/pull/355 to remove the duplicate `octavia_kuttl_run` target
